exercises/G-numbers/exercise.js

```diff
@@ -1 +1,19 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+let name = 'My nam is Ali. Number of students is :';
+let add = 'Number of mentors are :';
```

Comment:
Hey Ali , great job on this assignment, by looking at your code i realize how can i improve mine, so thank you a lot!

Reply:
Thank you for your feedback, it makes me stronger.